packages/expo-sms/android/src/main/java/expo/modules/sms/SMSModule.java

```diff
@@ -51,24 +52,13 @@ public void setModuleRegistry(ModuleRegistry moduleRegistry) {
 
   @ExpoMethod
   public void sendSMSAsync(final ArrayList<String> addresses, final String message, final Promise promise) {
-    if (!getContext().getPackageManager().hasSystemFeature(PackageManager.FEATURE_TELEPHONY) &&
-        !getContext().getPackageManager().hasSystemFeature(PackageManager.FEATURE_TELEPHONY_CDMA)) {
-      promise.reject(ERROR_TAG + "_UNAVAILABLE", "SMS service not available");
-      return;
-    }
-
-    if (mPromise != null) {
+    if (pendingPromise != null) {
```

Comment:
IMHO, it is redundant. 
It should not matter whether TELEPHONY is available, as long as there is any app willing to send SMSes. 
The worst can happen now is to open SMS app which in turn will not be able to sens SMS. I believe that it is more consistent and what users should really expect, especially on emulators, which might not have TELEPHONY, but will usually have SMS app.

On the other hand, it can cause some inconsistencies comparing to current behavior, so it might not be a good idea to change it now. It is up to your advice.